pkg/kubelet/util/pluginwatcher/plugin_watcher.go

```diff
@@ -211,7 +211,7 @@ func (w *Watcher) traversePluginDir(dir string) error {
 			}
 			//TODO: Handle errors by taking corrective measures
 			if err := w.handleCreateEvent(event); err != nil {
-				klog.Errorf("error %v when handling create event: %s", err, event)
+				return fmt.Errorf("error %v when handling create event: %s", err, event)
```

Comment:
Actually we can't really return here, we need to keep walking and register other plugins that might exist in the dir tree.

The error handling will be taken care of by the new plugin registration controller that's in the works.

Reply:
Can you point me to the PR number for the new plugin registration controller ?

Reply:
After walking, if there is no other plugin that exists in the dir tree, error should be returned, right ?

Reply:
ping @bertinatto for question above.

Reply:
ping @bertinatto for question above.